modules/tinymce/src/plugins/searchreplace/main/ts/ui/Dialog.ts

```diff
@@ -208,6 +208,7 @@ const open = (editor: Editor, currentSearchState: Cell<Actions.SearchState>): vo
     },
     onAction: (api, details) => {
       const data = api.getData();
+      toggleNotFoundAlert(false, api);
```

Comment:
this line is making fail the test in:
modules/tinymce/src/plugins/searchreplace/test/ts/browser/SearchReplacePrevNextTest.ts

Reply:
Fixed! Refreshing the dialog on every change was indeed a bit presumptuous.

Reply:
I thought that was weird, but couldn't see the wider context so let it ride 🤔